tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -2144,11 +2144,3 @@ def test_custom_measurement_size_filtering(self):
     @pytest.mark.xfail(reason="Having not supported")
     def test_having_condition(self):
         super().test_having_condition()
-
-    @pytest.mark.xfail(reason="Meta for project_id is wrong")
-    def test_project_id(self):
```

Comment:
Could you remind me why we want to remove these tests?

Reply:
There was a fault in the translation of meta values which led to project_id metadata being `string` instead of `UInt64`.

Reply:
Yeah, it's clear, but would you please let us know if your fix allows us to remove the `xfail` decorator? Or, as we agreed, please add new tests :)

Reply:
Should I write that it allows us to remove the `xfail`? I didn't get your comment